src/Stratis.Bitcoin.Features.BlockStore/StoreSettings.cs

```diff
@@ -85,7 +84,7 @@ public static void PrintHelp(Network network)
 
             builder.AppendLine($"-txindex=<0 or 1>              Enable to maintain a full transaction index.");
             builder.AppendLine($"-reindex=<0 or 1>              Rebuild chain state and block index from block data files on disk.");
-            builder.AppendLine($"-addressindex=<0 or 1>         Enable to maintain a full addresses index. Requires txindex to be enabled in order to work.");
+            builder.AppendLine($"-addressindex=<0 or 1>         Enable to maintain a full addresses index.");
```

Comment:
This is not entirely correct, it will maintain an address balance index.
Address index will imply you keep track of also spent UTXO (like explorers do).

Reply:
Indexer keeps track of all changes made to the address so I see no contradiction here.

Reply:
For clarity I would say it make more sense to add the word balance in the comment.